### ISSUE TITLE
Move all the head elements to its own file

### ISSUE THREAD

**Issue by SaraVieira** [axodotdev/oranda#75]
Right we have a `head.rs` but it does not contain all the head as when that was attempted an error from `axohtml` was thrown 

![image](https://user-images.githubusercontent.com/1051509/213194450-8e9b04c7-0b73-4ecf-b7cc-a9dbe6fe632b.png)

the error originates from https://github.com/axodotdev/axohtml/blob/remove-dodrio/macros/src/html.rs#L31

**Comment by ashleygwilliams** [comment]
could you open a branch/PR with this so I can poke at it @SaraVieira ?

**Comment by SaraVieira** [comment]
Sure! Here is a PR: https://github.com/axodotdev/oranda/pull/84

**Comment by ashleygwilliams** [comment]
this is blocked on axohtml macros crate issues

**Comment by ashleygwilliams** [comment]
closing in favor of a diff strategy- see notes on associated PR for more info
